kubernetes-client-api/src/main/java/io/fabric8/kubernetes/client/http/StandardHttpClient.java

```diff
@@ -72,7 +84,23 @@ public CompletableFuture<HttpResponse<AsyncBody>> consumeBytes(HttpRequest reque
       });
     }
 
-    return cf;
+    return withUpstreamCancelation(cf, AsyncBody::cancel);
+  }
+
+  static <V> CompletableFuture<HttpResponse<V>> withUpstreamCancelation(CompletableFuture<HttpResponse<V>> cf,
```

Comment:
```suggestion
  static <V> CompletableFuture<HttpResponse<V>> withUpstreamCancellation(CompletableFuture<HttpResponse<V>> cf,
```

Reply:
This looks to be a US vs British spelling difference https://www.grammarly.com/blog/canceled-vs-cancelled/ - since Java also uses the double L, I'll change it here also.